build/optaplanner-build-parent/pom.xml

```diff
@@ -35,6 +35,8 @@
     <version.org.freemarker>2.3.31</version.org.freemarker>
     <version.org.jdom>1.1.3</version.org.jdom>
     <version.org.jfree.jfreechart>1.5.3</version.org.jfree.jfreechart>
+    <version.org.openrewrite.recipe>1.11.0</version.org.openrewrite.recipe>
+    <version.org.slf4j>1.7.36</version.org.slf4j><!-- TODO keep in sync with quarkus-bom -->
```

Comment:
This is a PITA. Ideally, the quarkus-bom should declare jcl-over-slf4j too (it already defines slf4j-api).

 Or better yet, openrewrite-test should stop depending on stuff that depends on commons-logging.